src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -303,19 +303,28 @@ private void AppendException(Exception currentException, IEnumerable<ExceptionRe
                 }
             }
 
-            if (_customRenderingfunctions.TryGetValue(currentException.GetType(), out var renderer))
+            var exceptionType = currentException.GetType();
+            while (exceptionType != typeof(Exception))
```

Comment:
Please check if `_customRenderingfunctions.Count > 0` before starting all this logic.

Reply:
How do you use registered `_customRenderingfunctions` alone without getting the output from the configured `format`-property ? (Default = `message`).

Reply:
@snakefoot You can't do that currently. I don't think anyone would want to log specific properties and not message. I see custom renderer as an additional detail provider and if everyone needs a message NLog shouldn't force users to include message in every custom renderer

Reply:
I was trying to nudge you in the direction that you should add new enum value to ExceptionRenderingFormat

Reply:
>  I don't think anyone would want to log specific properties and not message.

one use case is to log both, but not in the same (database) column/field.